src/modlunky2/ui/trackers/category.py

```diff
@@ -78,6 +131,10 @@ def launch(self):
             config=self.modlunky_config.trackers.category,
         )
 
+    def config_update_callback(self):
```

Comment:
I'd move the `save()` call into here. I can't think of a scenario where we update the thread without saving the config.

A bonus is `CategoryModifiers` could take just the `CategoryTrackerConfig`